### ISSUE TITLE
Bug: Voltage readout doesn't display correctly <1V (AA lights)

### ISSUE THREAD

**Issue by jan-Sanku** [ToyKeeper/anduril#93]
this probably only applies to the D3AA at the moment. low voltage cells below 1V don't display correctly in the voltage readout.

at the voltage i tested it, **the light should've read out "098" (0.98V), but it only read out "98"** with a slightly longer pause in the place of the zero. other zeroes are fine (1.00V read as "100" in the same test)

it's only a very minor issue, at this voltage LVP is already kicking in, so it's clear that the battery is nearly depleted.

tested on `0161-2023-12-03-38`, hank's latest build i believe. i don't have a flashing adapter for the D3AA yet. i used a normal white eneloop AA.

**Comment by SammysHP** [comment]
I don't think this is a bug. Leading zeros are skipped in the current implement by design.

**Comment by jan-Sanku** [comment]
hm, i see. i think it's a bit strange to skip that zero though, it's not really "leading" if you consider there's a (implied) decimal point.

if it's not a bug, perhaps we can repurpose this issue into a feature suggestion?
